src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -731,7 +742,7 @@ export default {
 				const basketItemValues = data.shop?.basket?.items?.values ?? [];
 				this.itemsInBasket = basketItemValues.length ? basketItemValues.map(item => item.id) : [];
 
-				const credits = data.shop?.basket?.credits?.values;
+				const credits = data.shop?.basket?.credits?.values ?? [];
```

Comment:
guard against empty credits on the new basket after they've been spent or the page is visited without a promoCode